bin/bintray/upload

```diff
@@ -130,6 +134,7 @@ do
 		k) key="$OPTARG" ;;
 		P) publish=false ;;
 		o) override=true ;;
+		r) comp_force="$OPTARG" ;;
```

Comment:
@leandro-lucarella-sociomantic noted in discussion that this is surely the source of the bug with bintray upload: it should be `C)` here.